### ISSUE TITLE
Migrate to Azure.Search.Documents 11

### ISSUE THREAD

**Issue by xperiandri** [ssiltanen/Fasaani#1]

**Comment by ssiltanen** [comment]
Hi @xperiandri, thank you for opening this suggestion.

Migrating to Azure.Search.Documents is one of future goals for this library, however at the moment its documentation is still quite scarce, download counts low and apparently it has not yet "officially" replaced the old library (at least if you believe their release [documentation](https://azure.github.io/azure-sdk/releases/latest/index.html) which says this about the current library):

> Will be replaced by: Azure.Search.Documents".

I'll leave this issue open since this is a feature I will be looking into.

**Comment by xperiandri** [comment]
Yep. I usually go to fuget.org or GitHub to look at sources to figure out how to use it 😄

**Comment by xperiandri** [comment]
But since I've already written everything using it, it is not convenient to bring version 10 too
